content/en/docs/ops/deployment/performance-and-scalability/index.md

```diff
@@ -78,55 +78,56 @@ The latency, throughput, and the proxies' CPU and memory consumption are measure
 ### Sidecar CPU and memory usage
 
 Since the sidecar proxy performs additional work on the data path, it consumes CPU
-and memory. In Istio 1.22, a sidecar proxy consumes about 0.5 vCPU per 1000
-requests per second.
+and memory. In Istio 1.24, with 1000 http requests per second containing 1 KB of payload each
+- a sidecar proxy with 4 worker threads consumes about 27% CPU and 61 MB of memory.
+- a waypoint proxy with 2 worker threads consumes about 41% CPU and 57 MB of memory
+- a ztunnel proxy consumes about 7% CPU and 12 MB of memory.
```

Comment:
Nit: this is odd on "Sidecar CPU and memory usage" section.

Reply:
Also this implies ambient uses ~2x CPU (7+41=48% vs 27%). Is the sidecar only looking at 1 sidecar? If so we should make that more clear

Reply:
Is ztunnel adding up both ztunnels or just 1?

Reply:
For ztunnel I got 40m CPU and 2Mb memory each so hopefully that is adding them

Reply:
These are all per sidecar/ztunnel. Will clarify.